app/javascript/css/main.css.scss

```diff
@@ -23,7 +23,8 @@ a {
 .filter_block {
   position: absolute;
   background-color: #00000078;
-  height: 100%;
+  height: auto !important;
+  min-height: 100%;
```

Comment:
might be better one is to use `100vh`? simply idea

Reply:
> 

looks better, minus one additional property in another class